src/routes/user.js

```diff
@@ -22,4 +25,13 @@ router.patch('/:id', validateAuthentication, validateTeacherRole, updateById)
 router.patch('/update/:id', validateAuthentication, updateProfile)
 router.post('/:id/note', validateAuthentication, createNote)
 router.delete('/notes/:id', validateAuthentication, deleteNote)
+router.get('/:id/notes', validateAuthentication, getAllNotes)
+router.get('/notes/:id', validateAuthentication, getNoteById)
```

Comment:
Arguably this doesn't need to belong in the user router, and can live separately in a notes router. 

`/notes/:id` rather than `/user/notes/:id`

Thoughts?

Reply:
I agree, would it be better to move the getNoteById and the deleteNote to their own router, but keep the getAllNotes and createNote in the user router?